account-integrations/safe/test/hardhat/SafeBlsPlugin.test.ts

```diff
@@ -95,13 +87,14 @@ describe("SafeBlsPlugin", () => {
     ]);
 
     const deployedAddress = await calculateProxyAddress(
-      safeProxyFactory as any,
+      safeProxyFactory,
```

Comment:
Being able to remove `as any` is linked to the use of `*__factory.connect(addr, runner)` over `ethers.getContractFactory(...).attach(addr).connect(runner)` discussed elsewhere.